starter-core/src/main/java/io/micronaut/starter/feature/aws/Cdk.java

```diff
@@ -128,7 +127,7 @@ public void apply(GeneratorContext generatorContext) {
                         Template.DEFAULT_MODULE,
                         generatorContext.getBuildTool().isGradle() ? "build/libs" : "target",
                         generatorContext.getFeatures().hasFeature(AwsLambda.class) ? "micronaut-function" : null,
-                        generatorContext.getFeatures().hasFeature(AwsApiFeature.class) ? "micronaut-function-api" : null,
```

Comment:
I think we should leave `micronaut-function-api` since this is the resource name.